WooCommerce/Classes/ViewRelated/Blaze/BlazeCampaignList/BlazeCampaignItemView.swift

```diff
@@ -102,6 +102,7 @@ struct BlazeCampaignItemView: View {
                 .stroke(Color(uiColor: .separator), lineWidth: Layout.strokeWidth)
         }
         .padding(Layout.strokeWidth)
+        .contentShape(Rectangle())
```

Comment:
This fixes the issue when tapping on the campaign item doesn't work consistently.